src/components/ModalBox/BoxTitle.js

```diff
@@ -18,55 +18,50 @@
     51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
 */
 import * as React from 'react';
-import styled, { withTheme } from 'styled-components/native';
-
-import type { Theme } from 'models/Theme';
+import styled from 'styled-components/native';
 
 // components
 import { MediumText } from 'components/Typography';
 import IconButton from 'components/IconButton';
 
 // utils
 import { fontSizes, spacing } from 'utils/variables';
-import { getThemeColors } from 'utils/themes';
 
 type Props = {
   title: string,
   onPressClose: () => void,
-  theme: Theme,
 };
 
 const Wrapper = styled.View`
+  align-items: flex-start;
   display: flex;
   width: 100%;
+  padding: 0;
```

Comment:
Technically, there's no need to set it to 0. It is 0 by default :)

Reply:
Nice, removed.